sdk/ml/azure-ai-ml/azure/ai/ml/_internal/entities/_input_outputs.py

```diff
@@ -106,10 +106,18 @@ def _from_base(cls, _input: Union[Input, Dict]) -> Optional["InternalInput"]:
 
 
 class InternalOutput(Output):
-    def __init__(self, *, datastore_mode=None, **kwargs):
+    def __init__(self, *, datastore_mode=None, is_link_mode=None, **kwargs):
         self.datastore_mode = datastore_mode
+        self.is_link_mode = is_link_mode
```

Comment:
has it been added to corresponding Schema?

Reply:
It has been added to the InternalOutputSchema